modules/core/test/v2/unit/coins/utxo/coins.ts

```diff
@@ -0,0 +1,26 @@
+/**
+ * prettier
+ */
+import 'should';
+import * as utxolib from '@bitgo/utxo-lib';
+import { utxoCoins } from './util';
+
+describe('coins', function () {
+  it('getUtxoCoins', function () {
+    utxoCoins.map(c => [c.getChain(), utxolib.coins.getNetworkName(c.network)]).should.eql([
+      ['bch', 'bitcoincash'],
+      ['bsv', 'bitcoinsv'],
+      ['btc', 'bitcoin'],
+      ['btg', 'bitcoingold'],
+      ['dash', 'dash'],
+      ['ltc', 'litecoin'],
+      ['tbch', 'bitcoincashTestnet'],
+      ['tbsv', 'bitcoinsvTestnet'],
+      ['tbtc', 'testnet'],
+      ['tdash', 'dashTest'],
+      ['tltc', 'litecoinTest'],
+      ['tzec', 'zcashTest'],
+      ['zec', 'zcash'],
+    ]);
+  });
```

Comment:
```suggestion
  it('utxoCoin chains', function () {
    utxoCoins.map(c => c.getChain()).should.eql(
        ['bch', 'bsv', 'btc', 'btg', 'dash', 'ltc', 'tbch', 'tbsv', 'tbtc', 'tdash', 'tltc', 'tzec', 'zec']
    );
  });

  it('utxolib.coins.getNetworkName', function () {
    utxoCoins.map(c => utxolib.coins.getNetworkName(c.network)).should.eql([
      'bitcoincash',
      'bitcoinsv',
      'bitcoin',
      'bitcoingold',
      'dash',
      'litecoin',
      'bitcoincashTestnet',
      'bitcoinsvTestnet',
      'testnet',
      'dashTest',
      'litecoinTest',
      'zcashTest',
      'zcash',
    ]);
  });```

Reply:
I actually wanted to see them side by side

Reply:
Fair -- then just give it a long descriptive `it` like "verifies getUtxoCoins chains and network names".